packages/api/src/env.js

```diff
@@ -1,15 +1,32 @@
-/* global MAGIC_SECRET_KEY FAUNA_ENDPOINT FAUNA_KEY SALT CLUSTER_BASIC_AUTH_TOKEN CLUSTER_API_URL */
+/* global MAGIC_SECRET_KEY FAUNA_ENDPOINT FAUNA_KEY SALT CLUSTER_BASIC_AUTH_TOKEN CLUSTER_API_URL SENTRY_DSN */
+import Toucan from 'toucan-js'
 import { Magic } from '@magic-sdk/admin'
 import { DBClient } from '@web3-storage/db'
 import { Cluster } from '@nftstorage/ipfs-cluster'
 
+import pkg from '../package.json'
+
 /** @typedef {{ magic: Magic, db: DBClient, SALT: string }} Env */
 
 /**
- * @param {Request} _
+ * @param {Request} req
  * @param {Env} env
+ * @param {FetchEvent} event
  */
-export function envAll (_, env) {
+export function envAll (_, env, event) {
+  console.log('sentry', env.SENTRY_DSN || SENTRY_DSN)
+  env.sentry = new Toucan({
+    dsn: env.SENTRY_DSN || SENTRY_DSN,
+    event,
+    allowedHeaders: ['user-agent'],
+    allowedSearchParams: /(.*)/,
+    debug: false,
+    rewriteFrames: {
+      root: '/'
+    },
+    // version: env.VERSION || VERSION, // TODO: version endpoint?
+    pkg
+  })
```

Comment:
Please can sentry be optional? We don't need it when developing.

Reply:
sure!